src/components/ScholarshipList.jsx

```diff
@@ -84,11 +86,10 @@ function ScholarshipList({ noResultsNode, listFn }) {
 
 ScholarshipList.propTypes = {
   listFn: PropTypes.func,
-  noResultsNode: PropTypes.node,
+  noResultsNode: PropTypes.node.isRequired,
```

Comment:
You could make this optional, default to `undefined` and conditionally render the Typography element with the translation.